### DISCUSSION TITLE
Old Outlook SMTP server?

### DISCUSSION THREAD

**Discussion by gobbledygoober** [berrystephenw/OctoText#93] [General]
Per https://support.microsoft.com/en-us/office/pop-imap-and-smtp-settings-for-outlook-com-d088b986-291d-42b8-9564-9c414e2aa040 the Outlook.com SMTP server is (now?) smtp-mail.outlook.com

I discovered this while troubleshooting Test message issues. 

Wanted to let you know in case you wanted to update your README.md

**Comment by gobbledygoober** [comment]
After fixing my (unrelated) issue, I tested the server mentioned in the documentation, smtp.office365.com, and it also appears to work.

**Reply by berrystephenw** [reply]
Thanks - I just checked the settings for my account and it still says smtp.office365.com. Microsoft appears to have a bunch of aliases for their SMTP servers especially if you are outside the US. I tend to default to what is reported in the users email settings instead of trying to keep up with changes from Microsoft (or other services):
![image](https://user-images.githubusercontent.com/44931130/124459917-b5195c80-dd5c-11eb-8b06-cdeb7a5edb17.png)

Thanks for letting me know!

**Reply by gobbledygoober** [reply]
I had only viewed the Outlook settings on a mobile device and hadn't seen that settings screen -- perhaps it's only available on a desktop view. 

After a week of using the settings I reported, I suddenly began receiving authentication errors from OctoText. 

I had to "verify my account" in Outlook, and then restored access to the email account, but still got authentication errors from the plugin. 

Afterwards, I reverted to the default SMTP server your suggested, and the problem went away 🤔

**Reply by berrystephenw** [reply]
This doesn't surprise me, Outlook tries to be smart about spammers and limits the numbers of emails sent from new accounts. It happened to me when I was developing the plug-in but once I verified the account I haven't had a problem.
